deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/iterator/BertIterator.java

```diff
@@ -527,6 +651,15 @@ public Builder sentenceProvider(LabeledSentenceProvider sentenceProvider) {
             return this;
         }
 
+        /**
+         * Specify the source of the data for classification on sentence pairs.
+         * use case, the labels will be ignored.
```

Comment:
> use case, the labels will be ignored.

Not sure what this means

Reply:
This is from the non pair sentence provider. If sentence provider is given and task is unsupervised labels will be ignored. This was existing docs.

Reply:
> This was existing docs.

Let's cut it from here and the original location too then. It doesn't make sense.

Reply:
Done.